docs/wiki/installation/cli-flags.md

```diff
@@ -90,15 +90,13 @@ The level limits are as follows:
 Memory: default 200M, restrictive 100M
 CPU: default 10% (for 12 seconds), restrictive 5% (for 6 seconds)
 
-The normal level allows for 10 restarts if the limits are violated. The restrictive allows for only 4, then the service will be disabled. For both there is a linear backoff of 5 seconds, doubling each retry.
-
```

Comment:
There's no code that enforces this. (Maybe this was the case in the past and this is a residue.)